randovania/data/json_data/prime3.txt

```diff
@@ -951,9 +984,19 @@ Asset id: 13201618045565515280
 > Dock to Auxiliary Lift; Heals? False
   * Normal Door to Auxiliary Lift/Dock to Port Observation Deck
   > Dock to Xenoresearch Lab
-      Seeker Missile
+      After Port Observation Deck Seeker Wall
   > Dock to Junction A
-      Energy Cells Total
+      After Port Observation Deck Door Lock
+  > Event - Energy Cell Door Lock
+      Energy Cells Total ≥ 3
+  > Event - Seeker Wall
+      Seeker Missile
+
+> Event - Energy Cell Door Lock; Heals? False
+  * Event Port Observation Deck Door Lock
+
+> Event - Seeker Wall; Heals? False
+  * Event Port Observation Deck Seeker Wall
 
 ----------------
 Control Room Access
```

Comment:
Not commenting here, but for below. Cliffside Airdock, add node called "Samus Ship", with "Heals?" checked, and have it connect to the door as "Trivial"